hack/.golint_failures

```diff
@@ -6,9 +6,6 @@ cmd/kubeadm/app/apis/kubeadm/v1beta2
 pkg/apis/abac/v0
 pkg/apis/abac/v1beta1
 pkg/apis/admission
-pkg/apis/admissionregistration/v1
-pkg/apis/admissionregistration/v1beta1
-pkg/apis/admissionregistration/validation
```

Comment:
the validation package can be resolved, but the defaulting functions in the v1/v1beta1 packages should be left as-is

Reply:
updated.